tests/installation/partitioning.pm

```diff
@@ -21,10 +21,21 @@ sub run() {
 
     assert_screen 'partitioning-edit-proposal-button', 40;
 
+    if (check_screen ("install-from-iso-wrong-hdd")) {
```

Comment:
I'm also thinking about completely dropping the needles and depend this on the ISO_FROM_EXTERNAL_HDD variable